cypress.json

```diff
@@ -1,3 +1,5 @@
 {
-  "baseUrl": "http://localhost:3001"
+  "baseUrl": "http://localhost:3001",
+  "viewportWidth": 1280,
+  "viewportHeight": 720
```

Comment:
Is this pixels? Seems small, what's the reason?

Reply:
Default is 1000x660, so this is bigger, good aspect ratio when styling using cypress testing